var/spack/repos/builtin/packages/py-importlib/package.py

```diff
@@ -13,3 +13,5 @@ class PyImportlib(PythonPackage):
     pypi = "importlib/importlib-1.0.4.zip"
 
     version('1.0.4', sha256='b6ee7066fea66e35f8d0acee24d98006de1a0a8a94a8ce6efe73a9a23c8d9826')
+
+    depends_on('python@2.7.0:2.7.99')
```

Comment:
```suggestion
    depends_on('python@:2.6.999,3.0.0:3.0.999', type=('build', 'run'))
```
This is a backport of `importlib` from Python 2.7 to older versions of Python (2.6 and older, as well as 3.0, which came out before importlib).

Also, can you remove the `importlib` dependency from `py-fava`? It isn't needed.